handlers/main.yml

```diff
@@ -1,11 +1,18 @@
 ---
-- name: restart kernel_settings services
-  service:
-    name: "{{ item }}"
-    state: restarted
-  loop: "{{ __kernel_settings_services }}"
```

Comment:
What if there have been changes, but they do not require a reboot? Shouldn't you keep the "restart" task in this case?

Reply:
> What if there have been changes, but they do not require a reboot? Shouldn't you keep the "restart" task in this case?

It wasn't necessary to restart the service in order to apply the changes, all I need to do is call `tuned-adm profile ACTIVE_PROFILE` - https://github.com/linux-system-roles/kernel_settings/pull/4/files#diff-2444ad0870f91f17ca6c2a5e96b26823R93

And - in order to match the semantics of similar modules such as `sysctl`, the changes are not applied in the handler, they are applied immediately - this meets the expectations of tests such as https://github.com/linux-system-roles/kernel_settings/pull/7/files#diff-0363d71adca523f53e6765d01762b5c0R44 that expect the values are applied immediately after calling the role.

Reply:
Ah, I see this is explained in the commit message, sorry.